codewof/templates/users/user_detail.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}User: {{ user.get_full_name }}{% endblock %}
+
+{% block content %}
+  <div class="container">
+
+    <div class="row">
+      <div class="col-sm-12">
+        <h1>{{ user.get_full_name }}</h1>
+
+        <p><strong>Date joined</strong>: {{ user.date_joined }}</p>
+        <p><strong>Points earned</strong>: {{user.profile.points}}</p>
```

Comment:
for consistency add a space before/after inside the curly braces (also line 20, 22)